### PR TITLE
Logout at default scope allows full logout fixing #4780

### LINKED ISSUE #4780: Using the OneClick service, doesn't detect a logged in user
When using the OneClick (https://cartodb.com/open-in-cartodb/) service for an external dataset, OneClick does not detect an already authenticated CartoDB user. This forces the user to log in to CartoDB a second time.

Discussion here:
https://github.com/geoblacklight/geoblacklight/pull/341#issuecomment-126110096

### PR DESCRIPTION
**Pull request by juanignaciosl**
@Kartones CR this, please. Fixes #4780 (not on his own, needs Central side).

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5496/))

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -48,7 +48,9 @@ def create
   end
 
   def destroy
+    # Destroy session on both scopes: username and default
     logout(CartoDB.extract_subdomain(request))
+    logout
```
Comment (Kartones) [inline app/controllers/sessions_controller.rb]: what about `logout if authenticated?` but not sure if will fail if not authenticated with default scope

**Comment by Kartones** [comment]
:+1:

**Comment by juanignaciosl** [comment]
Retest this, please.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5502/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5510/))

**Pull request merged**